ui/pages/nightlies.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+import { request, gql } from 'graphql-request';
+
+import { Grid, IconButton, Stack, Tooltip, Typography } from '@mui/material';
+import { Circle } from '@mui/icons-material';
+
+const buildStatusColorMap = {
+  FAILED: 'error',
+  RUNNING: 'warning',
+  SUCCESS: 'success',
+  default: 'warning.dark' // WTF is going on colour!
+};
+
+const buildStatusColor = status => buildStatusColorMap[status] || buildStatusColorMap.default;
+
+const BuildStatus = ({ nightlies }) => {
+  const openNightlyClick = item => window.open(`https://gitlab.com${item.path}`, '_newtab');
+
+  return (
+    <>
+      <Typography variant="h4">Nightlies</Typography>
+      <Stack>
```

Comment:
Could we have the entire `Stack` stand out from the rest of the page a little bit? Like giving it a border or something? :thinking: